.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build-and-test:
+    working_directory: ~/ITKTextureFeatures-build
+    docker:
+      - image: insighttoolkit/module-ci:latest
+    steps:
+      - checkout:
+          path: ~/ITKTextureFeatures
+      - run:
+          name: Configure
+          command: |
+            cmake \
+              -G Ninja \
+              -DITK_DIR:PATH=/ITK-build \
+              -DBUILD_TESTING:BOOL=ON \
+              -DBUILDNAME:STRING=External-ITKTextureFeatures-${CIRCLE_BRANCH}-${CIRCLE_BUILD_NUM} \
+                ~/ITKTextureFeatures
+      - run:
+          name: Build and Test
+          no_output_timeout: 1.0h
```

Comment:
We can add

```
          environment:
            CTEST_BUILD_FLAGS: "-j 5"
```

to prevent ninja from attempting to use too spawn too many build processes, which is eating up all the memory.